core/tests/puppeteer-acceptance-tests/user-utilities/translation-admin-utils.js

```diff
@@ -0,0 +1,182 @@
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Translation admin users utility file.
+ */
+
+const baseUser = require(
+  '../puppeteer-testing-utilities/puppeteer-utils.js');
+const testConstants = require(
+  '../puppeteer-testing-utilities/test-constants.js');
+const ContributorDashboardAdminUrl =
+  testConstants.URLs.ContributorDashboardAdmin;
+const { showMessage } = require(
+  '../puppeteer-testing-utilities/show-message-utils.js');
+
+const translationRightValue = 'string:translation';
+const usernameMethodValue = 'username';
+const roleMethodValue = 'role';
+
+// "View Contributor Dashboard Users" form elements.
+const viewContributorMethodSelect =
+  'select#view-contributor-method-select';
+const viewContributerUsernameInput =
+  'input#view-contributor-username-input';
+const viewContributorCategorySelect =
+  'select#view-contributor-category-select';
+const viewContributorLanguageSelect =
+  'select#view-contributor-language-select';
+const viewContributorSubmitButton =
+  'button#view-contributor-submit-button';
+const viewContributorLanguageResult =
+  '.e2e-test-translation-reviewer-language';
+const viewLanguageRoleUserResult =
+  '.e2e-test-reviewer-roles-result';
+
+// "Add Contribution Rights" form elements.
+const addContributorUsernameInput =
+  'input#add-contribution-rights-user-input';
+const addContributonRightsCategorySelect =
+  'select#add-contribution-rights-category-select';
+const addContributonRightsLanguageDropdown =
+  'select#add-contribution-rights-language-select';
+const addontributionRightsSubmitButton =
+   'button#add-contribution-rights-submit-button';
+
+// "Remove Contribution Rights" form elements.
+const revokeContributorUsernameInput =
+  'input#revoke-contribution-rights-user-input';
+const revokeContributonRightsCategorySelect =
+  'select#revoke-contribution-rights-category-select';
+const revokeContributonRightsLanguageSelect =
+  'select#revoke-contribution-rights-language-select';
+const revokeContributionRightsSubmitButton =
+  'button#revoke-contribution-rights-submit-button';
+
+module.exports = class TranslationAdmin extends baseUser {
+  /**
+   * Function for navigating to the contributor dashboard admin page.
+   */
+  async navigateToContributorDashboardAdminPage() {
+    await this.goto(ContributorDashboardAdminUrl);
+  }
+
+  /**
+   * Function for assigning a translation right to a user.
+   * @param {string} username - the username of the user.
+   * @param {string} languageCode - the language code of the language to assign.
+   */
+  async assignTranslationRightsToUserWithLanguageCode(username, languageCode) {
```

Comment:
```suggestion
  async addTranslationLanguageReviewRights(username, languageCode) {
```
"Add" matches more closely with the UI and "ToUserWithLanguageCode is a bit redundant with the function params.

Reply:
Done.